source/getting-started/connection.txt

```diff
@@ -0,0 +1,622 @@
+================
+Connection Guide
+================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+This guide shows you how to connect to MongoDB with the Node.js driver using
+different connection mechanisms as well as how to configure them.
+
+When you connect to a MongoDB server, you provide your credentials and
```

Comment:
Thanks for that info! I can definitely use **instance** or **deployment**. I just wanted to be specific to avoid confusion between client/server and instance/deployment works fine.

Reply:
FAIR! People have historically had both client/server confusion and (mongodb) server / (host) server confusion, hence the instance/deployment preference, but it's just convention at this point :)